app/test_main.py

```diff
@@ -1,3 +1,53 @@
+import pytest
+from typing import Any
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,expected_ages",
+    [
+        (14, 14, [0, 0]),
+        (23, 23, [1, 1]),
+        (15, 15, [1, 1]),
+        (24, 24, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17])
```

Comment:
Add test cases for negative numbers, zero numbers, and different numbers for cat and dog age as mentioned [in the checklist](https://github.com/sachavskyi/py-cat-and-dog-years/blob/develop/checklist.md#clean-code).